pkg/linter/linter.go

```diff
@@ -194,6 +194,11 @@ var linterMap = map[string]linter{
 		Explain:         "Move binary to /usr/bin",
 		defaultBehavior: Require,
 	},
+	"cudaruntimelib": {
+		LinterFunc:      allPaths(cudaDriverLibLinter),
+		Explain:         "CUDA driver-specific libraries should be passed into the container by the host. If this library is needed for build-time linking or ldd-check tests, please include a stub library during build or test instead.",
```

Comment:
Personally, I don't know if this is enough information for me to fix the warning. Is there further documentation or an example to which you can point?

Reply:
i've added more specific instructions